scheduler/graphql_mid/schema.py

```diff
@@ -44,7 +44,8 @@ async def load_sources_files(self, files_input: UseFilesSourceInput) -> SourceFi
                 gmos_fpu = await files_input.gmos_fpus.read()
                 gmos_gratings = await files_input.gmos_gratings.read()
 
-                loaded = sources.use_file(service,
+                loaded = sources.use_file(files_input.sites,
+                                          service,
```

Comment:
I think we're going to have to include faults and eng tasks in here now.
I'm not sure about calendar, since this is a Google sheet. Let's figure out what we need to do here about that.